### PR TITLE
lesson-1.5 well done

### PR DESCRIPTION
**Pull request by zemiarsi**

### CODE REVIEW

File: lesson_1.1/src/App.js [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = useState(
+    localStorage.getItem ('savedTodoList')? JSON.parse(localStorage.getItem('savedTodoList')) : [] 
+  );
+
+useEffect(() => {
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:11]: lines 11 and 13 should be indented one place

File: lesson_1.1/src/App.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+  
+return [todoList, setTodoList];
+
+}
+
 function App() {
-  const [todoList, setTodoList]= useState([]);
+  const [todoList, setTodoList]= useSemiPersistentState();
+
 const addTodo = (newTodo)=> {
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:24]: addTodo function should be indented one space

**Review by kaytijackson97 — COMMENTED** [review]
Well done! you've definitely understood making custom hooks and applying them with your local storage. fix a couple indentations and I would be more than happy to accept this pull request at work. keep up the excellent work!

**Comment by zemiarsi** [comment]
Good Morning Kayti,

Thanks,  I love this lesson  much more

**Pull request merged**